From 75f4e59756e501eea33b611fa39c86ffec47e14b Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@totoro.usersys.redhat.com>
Date: Tue, 5 Feb 2013 10:52:03 -0800
Subject: [PATCH 28/33] Ticket #572 - PamConfig schema not updated during
 upgrade

Bug description: PAM passthrough schema was not upgraded in the
upgrade from 389-ds-base-1.2.10.2 to 389-ds-base-1.2.11.15.

Fix description: This patch adds 60pam-plugin.ldif to the upgrade
schema file list.

Reviewed by Nathan (Thank you!!)
(cherry picked from commit ac6b2c94bb32fc47870a56e8f7fc109ea81a3f47)
---
 ldap/admin/src/scripts/60upgradeschemafiles.pl |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ldap/admin/src/scripts/60upgradeschemafiles.pl b/ldap/admin/src/scripts/60upgradeschemafiles.pl
index c308716..1208f4d 100644
--- a/ldap/admin/src/scripts/60upgradeschemafiles.pl
+++ b/ldap/admin/src/scripts/60upgradeschemafiles.pl
@@ -11,7 +11,7 @@ sub runinst {
 
     # these schema files are obsolete, or we want to replace
     # them with newer versions
-    my @toremove = qw(00core.ldif 01core389.ldif 01common.ldif 02common.ldif 05rfc2247.ldif 05rfc4523.ldif 05rfc4524.ldif 06inetorgperson.ldif 10presence.ldif 28pilot.ldif 30ns-common.ldif 50ns-directory.ldif 60mozilla.ldif 60sudo.ldif);
+    my @toremove = qw(00core.ldif 01core389.ldif 01common.ldif 02common.ldif 05rfc2247.ldif 05rfc4523.ldif 05rfc4524.ldif 06inetorgperson.ldif 10presence.ldif 28pilot.ldif 30ns-common.ldif 50ns-directory.ldif 60mozilla.ldif 60pam-plugin.ldif 60sudo.ldif);
 
     # these hashes will be used to check for obsolete schema
     # in 99user.ldif
-- 
1.7.7.6

